backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gdx-agreements-tracker-api",
-  "version": "0.7.0",
+  "version": "0.9.0",
```

Comment:
one of these days we should find a way to prevent these version bumps on each commit

Reply:
Agreed, this is not working the way we envisioned.